src/renderer/components/settings/Information.tsx

```diff
@@ -47,15 +47,13 @@ export class LanguageSettings extends React.Component<Props, States> {
 
     public async componentDidMount() {
         const { locale } = this.props;
-
-        try {
-            let fileContent = await promisify(readFile)(`src/resources/information/${locale}.md`, {encoding: "utf8"});
+        if (info[locale]) {
+            let fileContent = info[locale] as unknown as string;
```

Comment:
Now that `info` is declared as `const info: { [key: string]: typeof import("*.md") }`, can this line become `let fileContent = info[locale]`? (why do we need explicit type coercion `as unknown as string`?) If not, can we just replace `typeof import("*.md")` with `string`?